src/types/code.ts

```diff
@@ -1,5 +1,5 @@
 export type Code = {
-  system: 'SNOMED-CT' | 'RxNorm' | 'LOINC' | 'NUBC' | 'DICOM-DCM' | 'DICOM-SOP';
+  system: 'SNOMED-CT' | 'RxNorm' | 'LOINC' | 'NUBC' | 'DICOM-DCM' | 'DICOM-SOP' | '';
```

Comment:
Are there potential repercussions for adding an empty string here? Wasn't sure how to resolve the code system.

Reply:
Since there is code that may set this to `''` it is necessary. So I think it is okay.